### DISCUSSION TITLE
經營權之爭陷法律戰 龍邦提告後 泰山也按鈴申告

### DISCUSSION THREAD

**Discussion by Floriaand** [1101-GS4719-C/Python-Community-GS4719-C#906] [General]
這個事件一開始是龍邦想收購泰山，就買了很多他的股票，最後泰山為了要讓龍邦放棄收購，就將他們最值錢的全家的股份賣掉了，那龍邦就想告泰山罔顧股東的權益就將股票賣掉。我覺得這次的事件後，不管有沒有被收購對泰山來說傷害都很大，因為它賣掉了他們最賺錢的東西。
網址：https://www.google.com/amp/s/amp-news.cnyes.com/news/id/5039288

**Comment by CPCpyt** [comment]
但比起獲利，我想對管理的家族來說泰山企業本身才是真的的靠山，
雖然有點斷尾求生那味，但是比起未來持續穩定，在當下時存活才是更重要的。
